openforcefield/typing/engines/smirnoff/parameters.py

```diff
@@ -1166,6 +1220,35 @@ def __init__(self, **kwargs):
     def __init__(self, **kwargs):
         super().__init__(**kwargs)
 
+    def check_handler_compatibility(self,
+                                    other_handler):
+        """
+        Checks whether this ParameterHandler encodes the same physics as another ParameterHandler. This is
```

Comment:
I wouldn't say "the same physics", but at least "compatible physics".

Reply:
Good call. Added.